internal/lsp/server.go

```diff
@@ -1610,6 +1739,13 @@ func (l *LanguageServer) handleInitialize(
 		},
 	}
 
+	// Since evaluation requires some client side handling, this can't be supported
+	// purely by the LSP. Clients that are capable of handling the code lens callback
+	// should be added here though.
+	if l.clientIdentifier == clients.IdentifierVSCode {
+		initializeResult.Capabilities.CodeLensProvider = &types.CodeLensOptions{}
```

Comment:
Perhaps to come later, but something we can write to an result.json instead perhaps, but still show the command. (as discussed)

Reply:
Yep, that'll come next!